src/main/java/org/humancellatlas/ingest/process/ProcessRepository.java

```diff
@@ -23,6 +24,8 @@
     Page<Process> findBySubmissionEnvelopesContaining(SubmissionEnvelope submissionEnvelope,
             Pageable pageable);
 
+    Page<Process> findByInputBundleManifestsContaining(BundleManifest bundleManifest, Pageable pageable);
```

Comment:
could set this to not be exposed since it's a bit confusing to see both findByInputBundleUuid and findByInputBundleManifestsContaining in the search links